src/extensions/default/RemoteFileAdapter/main.js

```diff
@@ -1,46 +1,42 @@
 /*
- * Copyright (c) 2018 - present Adobe Systems Incorporated. All rights reserved.
+ * GNU AGPL-3.0 License
```

Comment:
Please change license to MIT, with copyright the line 
```
Modified Work Copyright (c) 2021 - present core.ai & brackets.io. All rights reserved.
Original work Copyright (c) 2018 - 2021 Adobe Systems Incorporated. All rights reserved.
```

Reply:
Fixed (hopefully).

Reply:
Wait a second- not fixed.

Reply:
Wait another second- fixed.

Reply:
No, wasn’t fixed, but is now.